vscode-dotnet-runtime-library/src/Acquisition/IDistroDotnetSDKProvider.ts

```diff
@@ -197,9 +197,60 @@ export abstract class IDistroDotnetSDKProvider {
             }
         }
 
+        if(availableVersions.length === 0 && !haveTriedFeedInjectionAlready)
+        {
+            // PMC is only injected and should only be injected for MSFT feed distros.
+            // Our check runs by checking the feature band first, so that needs to be supported for it to fallback to the preinstall command check.
+            const fakeVersionToCheckMicrosoftSupportStatus = '6.0.1xx';
```

Comment:
Is there chances that the version might be lower than '6.0.1xx'?

Reply:
That's a good question -- it looks like older versions that are now unsupported do support older versions of dotnet. But the 6.0 is really just a placeholder version, and it will get mapped to whatever is closest. Rereading this a month later, this code is clear but a bit more complicated than perhaps necessary, and the comment above appears to be meant to allude to the fact that 6.0 is just a placeholder to pass the unsupported check in `getDotnetVersionSupportStatus`